samples-test/samples/Shell/rename.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```

Since you're using the `.sh` extension for this sample, I think you want to use this shebang, too.